starter-core/src/main/java/io/micronaut/starter/feature/FeatureUtils.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature;
+
+import com.fizzed.rocker.RockerModel;
+import io.micronaut.starter.application.Project;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.template.RockerTemplate;
+
+public final class FeatureUtils {
```

Comment:
Do we need a new class here, can these methods not just go into `GeneratorContext`? Please delete this.

Reply:
I've removed this class and added the methods into `GeneratorContext`